tests/integration/test_orders_cli.py

```diff
@@ -654,6 +844,35 @@ def test_cli_orders_create_email(invoke, geom_geojson, order_description):
     assert sent_request == order_request
 
 
+def test_cli_orders_request_tools(invoke, geom_geojson,
+                                  write_to_tmp_json_file):
+    tools_json = [{'clip': {'aoi': geom_geojson}}, {'composite': {}}]
+    tools_file = write_to_tmp_json_file(tools_json, 'tools.json')
```

Comment:
@kevinlacaille what would you think about combining `geom_geojson` and `write_to_tmp_json_file` into a single pytest fixture? Something like this:

```suggestion
@pytest.fixture
def tools_file(geom_geojson, write_to_tmp_json_file):
    tools_json = [{'clip': {'aoi': geom_geojson}}, {'composite': {}}]
    return write_to_tmp_json_file(tools_json, 'tools.json')
    
def test_cli_orders_request_tools(invoke, tools_file):
```

Reply:
Could be a useful pattern for the cloud config files too?